app/assets/javascripts/components/DisplayLogicFormField.jsx

```diff
@@ -0,0 +1,56 @@
+class DisplayLogicFormField extends React.Component {
+  constructor(props) {
+    super();
+    this.state = props;
+    this.changeDisplayOption = this.changeDisplayOption.bind(this);
+    this.buildConditions = this.buildConditions.bind(this);
+    this.addCondition = this.addCondition.bind(this)
+  }
+
+  changeDisplayOption(event) {
+    let value = event.target.value
+
+    this.setState({display_if: value})
+  }
+
+  buildConditions(args) {
+    if(this.state.display_if != "always") {
+      return (
+        <div>
+          {this.state.display_conditions.map((props, index) => <ConditionsFormField key={index} {...props}/>)}
+          <button onClick={this.addCondition} type="button" className="btn add_condition">
```

Comment:
The test suite needs a way to find the button and since it contains the icon, you can't search for it in capybara by label value. The test I had been writing that actually added multiple conditions is no longer in here (ran into some insurmountable issues with getting capybara to find and pick the correct selects for the additional conditions), so technically it's extraneous right now, but you're likely to need either an ID or a class to identify that button with.

Reply:
k